bin/test-db

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+import os
+import sys
+import argparse
+import subprocess
+import time
+from argparse import RawTextHelpFormatter
+
+image_name = "singerio/mysql"
+image_tag = "5.7.29-binlog"
+
+def start_container(name):
+    START_COMMAND = """
+    sudo docker run -d --rm -e "MYSQL_ROOT_PASSWORD={password}" -e "MYSQL_DATABASE={database}" \
```

Comment:
Docker does have a python library if you want to avoid shelling out: https://docker-py.readthedocs.io/en/stable/ . Unlike docker-compose, docker is just a wrapper for calling rest endpoints against the local daemon. Personally I prefer to avoid the shell if I can, but it's up to you.

Reply:
We can try it next time. This is mostly incidental so I didn't want to start a new pattern.

Reply:
May actually look into this if we can make headway on this card today and have time.